tests/nightly/gpu/test_anti_scaling.py

```diff
@@ -0,0 +1,180 @@
+        """
+
+        precise_mode = False
+        # Turn this on to check the loss terms for TinyBERT-style distillation, which
+        # relies upon weights being initialized in a particular way. Won't work on
+        # CircleCI machines
+
+        torch.manual_seed(0)
+        np.random.seed(0)
+
+        opts_and_desired_losses = [
+            (
+                self.TRANSFORMER_OPT,
+                self.WIDE_DISTILLATION_OPT,
+                'DistillTransformerAgent',
+                False,
+                {
+                    'dec_hid_loss': 87.27,
+                    'enc_hid_loss': 0.8726,
+                    'enc_loss': 0.8726,
+                    'loss': 15.85,
+                    'pred_loss': 13.77,
+                },
+            ),
+            (
+                self.BART_OPT,
+                self.WIDE_DISTILLATION_OPT,
+                'DistillBartAgent',
+                False,
+                {
+                    'dec_hid_loss': 2.731,
+                    'enc_hid_loss': 0.0383,
+                    'enc_loss': 0.0383,
+                    'loss': 32.16,
+                    'pred_loss': 29.35,
+                },
+            ),
+            (
+                self.TRANSFORMER_OPT,
+                self.NARROW_DISTILLATION_DUMMY_LOSS_OPT,
+                'DistillNarrowTransformerAgent',
+                True,
+                {
+                    'dec_emb_loss': 1.625,
+                    'dec_hid_loss': 49.08,
+                    'dec_self_attn_loss': 3.609,
+                    'enc_dec_attn_loss': 29.65,
+                    'enc_emb_loss': 1.027,
+                    'enc_hid_loss': 0.5528,
+                    'enc_loss': 0.5478,
+                    'enc_self_attn_loss': np.inf,
+                    'loss': 11.41,
+                    'pred_loss': 9.238,
+                },
+            ),
+            (
+                self.BART_OPT,
+                self.NARROW_DISTILLATION_DUMMY_LOSS_OPT,
+                'DistillNarrowBartAgent',
+                True,
+                {
+                    'dec_emb_loss': 9.495,
+                    'dec_hid_loss': 0.6304,
+                    'dec_self_attn_loss': 514.2,
+                    'enc_dec_attn_loss': 137.6,
+                    'enc_emb_loss': 6.642,
+                    'enc_hid_loss': 0.05015,
+                    'enc_loss': 0.05089,
+                    'enc_self_attn_loss': 9.318,
+                    'loss': 11.39,
+                    'pred_loss': 8.918,
+                },
+            ),
+        ]
+        for (
+            model_opt,
+            distillation_opt,
+            model_name,
+            is_tinybert_style,
+            desired_losses,
+        ) in opts_and_desired_losses:
+            opt = {
+                **self.BASE_OPT,
+                **model_opt,
+                **distillation_opt,
+                'model': f'{self.DISTILLATION_MODEL_PREFIX}:{model_name}',
+                'num_examples': 1,
```

Comment:
Given that you just want to run through a single example, let's remove the dependency on data.

Instead of using eval model, roll a manual observe act loop with a hardcoded text. Will be faster and more reliable.

Reply:
You can get rid of the seeds too then.

For training, just set all the dropout parameters to 0.0 so it's deterministic.

Reply:
Yes, good point - will replace with hardcoded text. However, I'll still need the seeds because the distillation agents still perform random initialization of weights, especially the TinyBERT-style agents where you can't copy over the teacher weights